livesync_citadel/indico_livesync_citadel/schemas.py

```diff
@@ -0,0 +1,205 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2020 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from marshmallow import post_dump
+from tika import parser
+from webargs import fields
+
+from indico.core.db.sqlalchemy.links import LinkType
+from indico.core.marshmallow import mm
+from indico.modules.attachments.models.attachments import Attachment, AttachmentType
+from indico.modules.events import Event
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.contributions.models.subcontributions import SubContribution
+from indico.modules.events.notes.models.notes import EventNote
+from indico.web.flask.util import url_for
+
+
+class ACLSchema(object):
+    _access = fields.Method('_get_default_acl')
+
+    def _get_default_acl(self, *args):
+        """Return the object ACLs. More information here https://cern-search.docs.cern.ch/usage/permissions/"""
+        from indico_livesync_citadel.plugin import LiveSyncCitadelPlugin
+        default_acl =  LiveSyncCitadelPlugin.settings.get('search_owner_role')
```

Comment:
```suggestion
        default_acl = LiveSyncCitadelPlugin.settings.get('search_owner_role')
```

(run flake8, it should find such things)